apps/server/src/modules/system/controller/system.controller.ts

```diff
@@ -5,12 +5,12 @@ import { SystemFilterParams } from '@src/modules/system/controller/dto/system.fi
 import { SystemOauthResponseMapper } from '@src/modules/system/controller/mapper/system-oauth-response.mapper';
 import { SystemUc } from '../uc/system.uc';
 
-@ApiTags('System')
-@Controller('system')
+@ApiTags('Systems')
+@Controller('systems')
 export class SystemController {
 	constructor(private readonly systemUc: SystemUc) {}
 
-	@Get()
+	@Get('public')
 	@ApiOperation({ summary: 'Finds all systems.' })
 	@ApiResponse({ status: 200, type: SystemOauthResponse, description: 'Returns an SystemOauthResponse.' })
 	async find(@Query() filterParams: SystemFilterParams): Promise<SystemOauthResponse> {
```

Comment:
looking at the code, I find unsecured the fact that a string (for the type arg) is taken from form URL and simply passed along all the way to the repo
type could be an enum
also, any string should be sanitised